src/components/StripeCardInput/cardInput.js

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { CardElement, useElements, useStripe } from '@stripe/react-stripe-js';
+import { useTheme, useUniqueIdentifier } from '../../libs/hooks';
+import RenderIf from '../RenderIf';
+import Label from '../Input/label';
+import HelpText from '../Input/styled/helpText';
+import ErrorText from '../Input/styled/errorText';
+import StyledContainer from './styled/container';
+import StyledFakeDisabled from './styled/fakeDisabled';
+import getError from './helpers/getErrror';
+import getCardElementOptions from './helpers/getCardElementOptions';
+
+const CardInput = props => {
+    const {
+        label,
+        hideLabel,
+        bottomHelpText,
+        error,
+        className,
+        style,
+        disabled,
+        onChange,
+        onFocus,
+        onBlur,
+    } = props;
+    const stripe = useStripe();
+    const elements = useElements();
+    const cardElementId = useUniqueIdentifier('card-element');
+    const theme = useTheme().rainbow;
+    const cardElementOptions = useMemo(() => getCardElementOptions(theme, disabled), [
+        theme,
+        disabled,
+    ]);
+    const handleChange = event => {
+        if (stripe && elements) {
```

Comment:
is there onchange event happening before stripe and elements got ready?? if so, should we render nothing (null) until those object are ready?? I think we are being defensive here where we dont need it